### PR TITLE
Remove CLI Switches from Automated Test Suites and Catalog Installation

### PR DESCRIPTION
**Pull request by dubeejw**
- Remove Go/Python CLI test switches
- Remove all references to the Python CLI from the automated tests

PG1: 384

### CODE REVIEW

**Comment by rabbah** [comment]
Suggest you wait until the scala test rewrite is merged.

File: tests/src/common/WhiskProperties.java [hunk 1]
```diff
@@ -123,20 +123,6 @@ public static String getGoCLIPath() {
         return getGoCLIDir() + "/wsk";
     }
 
-    /**
-     * The path to the Python CLI directory.
-     */
-    public static String getPythonCLIDir() {
-        return whiskHome + "/bin";
```
Comment (rabbah) [inline tests/src/common/WhiskProperties.java]: i will miss you.

File: tests/src/whisk/core/cli/test/WskBasicUsageTests.scala [hunk 2]
```diff
@@ -100,30 +92,41 @@ class WskBasicCliUsageTests
...
-        val apihost = s"http://${WhiskProperties.getControllerHost}:${WhiskProperties.getControllerPort}"
-        val stdout = wsk.cli(Seq("--apihost", apihost, "property", "get", "--apibuild")).stdout
-        stdout should not { include regex ("Cannot determine API build") or include regex ("Unknown") }
-        stdout should include regex ("""(?i)whisk API build\s+201.*""")
+        val tmpwskprops = File.createTempFile("wskprops", ".tmp")
+        try {
+            val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())
+            val apihost = s"http://${WhiskProperties.getControllerHost}:${WhiskProperties.getControllerPort}"
+            wsk.cli(Seq("property", "set", "--apihost", apihost), env = env)
+            val rr = wsk.cli(Seq("property", "get", "--apibuild", "-i"), env = env, expectedExitCode = 3)
```
Comment (rabbah) [inline tests/src/whisk/core/cli/test/WskBasicUsageTests.scala]: why is this right?  line 123 sets the api host shouldn't 124 retrieve the build info?

File: tests/src/whisk/core/cli/test/WskBasicUsageTests.scala [hunk 3]
```diff
@@ -100,30 +92,41 @@ class WskBasicCliUsageTests
...
-        val wsk = new Wsk(usePythonCLI = true)
-        val rr = wsk.cli(Seq("--apihost", "xxxx.yyyy", "property", "get", "--apibuild"), expectedExitCode = ANY_ERROR_EXIT)
-        rr.stdout should { include regex ("Cannot determine API build") or include regex ("Unknown") }
-        rr.stdout should not include regex("""(?i)whisk API build\s+201.*""")
-        rr.stderr should not include ("https:///api/v1: http: no Host in request URL")
+        val tmpwskprops = File.createTempFile("wskprops", ".tmp")
+        try {
+            val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())
+            wsk.cli(Seq("property", "set", "-i", "--apihost", "xxxx.yyyy"), env = env)
+            val rr = wsk.cli(Seq("property", "get", "--apibuild", "-i"), env = env, expectedExitCode = 3)
```
Comment (rabbah) [inline tests/src/whisk/core/cli/test/WskBasicUsageTests.scala]: can we factor out `3` into a const encoded in test utils?

**Comment by rabbah** [comment]
don't squash new commits, if any.

**Comment by rabbah** [comment]
moved back to wip pending to address questions above - kick back to review when ready.

**Comment by dubeejw** [comment]
Will do.

**Comment by rabbah** [comment]
@dubeejw pg?

**Comment by dubeejw** [comment]
PG1 384, it's in the PR description.

**Comment by rabbah** [comment]
Rebase - there is a conflict.

**Comment by dubeejw** [comment]
@rabbah, rebase complete. PG1 423.

**Comment by rabbah** [comment]
Merged as https://github.com/openwhisk/openwhisk/commit/5efca26f33e3b57346a69fb98232283705e122d8.

**Pull request closed without merging**